controller/static/app/registry/addRegistry.controller.js

```diff
@@ -31,9 +31,9 @@
                 .success(function(data, status, headers, config) {
                     $state.transitionTo('dashboard.registry');
                 })
-            .error(function(data, status, headers, config) {
-                vm.error = data;
-            });
+                .error(function(data, status, headers, config) {
```

Comment:
Ah -- this change actually fixed the formatting.  The chained call to .error should be in line with the .success, right?